task/preflight/0.1/preflight.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: preflight
+spec:
+  description: >-
+    Scans container images for certification readiness
+  params:
+    - name: image-url
+      description: Image url to scan.
+  results:
+    - name: TEST_OUTPUT
+      description: Preflight pass or fail outcome.
+    - name: PFLT_OUTPUT
+      description: Preflight check summary.
+  steps:
+    - name: check-container
+      image: quay.io/opdev/preflight:stable@sha256:d990cbe8fb0db4dff0f7609403992f273f07d55f89dc7d2f046eb8f9c3e48110
+      args: ["check", "container", "$(params.image-url)"]
+      volumeMounts:
+        - name: pfltoutputdir
+          mountPath: /artifacts
+    - name: gather-pflt-results
+      image: quay.io/redhat-appstudio/appstudio-utils:cb1a8a07fa536d0f661990ff06e10954895d448a
```

Comment:
this image should have also digest, I just noticed it looked like a digest from distance but it's just a tag

Reply:
Sorry for the delay. I have updated it to a digest.